src/AElf.WebApp.Web/WebAppAbpServiceConvention.cs

```diff
@@ -16,7 +16,11 @@ public WebAppAbpServiceConvention(IOptions<AbpAspNetCoreMvcOptions> options) : b
         protected override string CalculateRouteTemplate(string rootPath, string controllerName, ActionModel action, string httpMethod,
             ConventionalControllerSetting configuration)
         {
-            
+            if (!controllerName.StartsWith("Net"))
```

Comment:
Cannot Attribute work?

Reply:
I can use AspNetCore.Mvc.RouteAttribute, but ABP will modify the method name in the route.

so,I will use [Route("api/blockChain/taskQueueStatus")]... at the next modification.

Reply:
ok,